keyboards/ergodox_ez/keymaps/lukaus/readme.md

```diff
@@ -0,0 +1,2 @@
+#Lukaus's keymap
```

Comment:
You need a space after the `#` here, and a blank line between this one and the next one.